plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/rule/QueryGroupFeatureType.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.rule;
+
+import org.opensearch.autotagging.Attribute;
+import org.opensearch.autotagging.AutoTaggingRegistry;
+import org.opensearch.autotagging.FeatureType;
+
+import java.util.Map;
+
+/**
+ * Represents a feature type specific to the query group feature
+ * @opensearch.experimental
+ */
+public class QueryGroupFeatureType implements FeatureType {
```

Comment:
These `QueryGroup` references should be updated to `WorkloadGroup` here and few other places in this PR